tasks/task05/user_factory.py

```diff
@@ -0,0 +1,14 @@
+def create_user(*args, age=42, **kwargs):
```

Comment:
Good, but use `first_name=` and `last_name=` instead of `*args` it will throw exception, when you don't pass this variables

Reply:
And good practice, when you check type in parameters.
Like this: `first_name:str, last_name:str, age:int = 42, **kwargs`. It doesn't allow to pass any 'bad' value.